test/fixtures/deploy-full-suite.fixture.ts

```diff
@@ -252,6 +252,7 @@ export async function deploySuiteWithModularCompliancesFixture() {
 
   const complianceProxy = await ethers.deployContract('ModularComplianceProxy', [context.authorities.trexImplementationAuthority.address]);
   const compliance = await ethers.getContractAt('ModularCompliance', complianceProxy.address);
+  await compliance.bindToken(context.suite.token.address);
```

Comment:
I think having this binding here breaks many other tests (many tests are failing once the `.only` has been removed).

Reply:
I would remove it from here

```suggestion
```

And add it to your specific test fixtures.